config/apidoc.php

```diff
@@ -87,4 +87,16 @@
             ],
         ],
     ],
+
+    /*
+     * Custom logo path. Will be copied during generate command.
+     *
+     * If you want to use this, please be aware of the following rules:
+     * - filename: logo.png
```

Comment:
I think the requirement on the filename can be removed, since they're providing the full path, including the filename.